build error: [E0599]: no method named `slice_to_string`

OS: macOS 10.14
Version: rustc 1.29.1

run:
```bash
cd rust
cargo build
```
output:
```bash
error[E0599]: no method named `slice_to_string` found for type `xi_rope::tree::Node<xi_rope::rope::RopeInfo>` in the current scope
   --> core-lib/src/editor.rs:705:42
    |
705 |                     let word = self.text.slice_to_string(start..end);
    |                                          ^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: Could not compile `xi-core-lib`.

To learn more, run the command again with --verbose.
```

Then I try to change `slice_to_string` to `slice_to_cow`, build success.

--- a/rust/core-lib/src/editor.rs
+++ b/rust/core-lib/src/editor.rs
@@ -702,7 +702,7 @@ impl Editor {
 
                 if start < end {
                     let interval = Interval::new_closed_open(start, end);
-                    let word = self.text.slice_to_string(start..end);
+                    let word = self.text.slice_to_cow(start..end);
 
                     // first letter is uppercase, remaining letters are lowercase
                     let (first_char, rest) = word.split_at(1);

